dimagi/pages/templates/pages/open_source/open_source_home.html

```diff
@@ -0,0 +1,54 @@
+{% extends "base.html" %}
+{% load i18n %}
+{% load dimagidotcom %}
+
+{% block title %}{% blocktrans %}
+  CommCare by Dimagi | Open Source Home
+{% endblocktrans %}{% endblock %}
+
+{% block description %}{% blocktrans %}
+  CommCare is a leading open source mobile data collection platform that is supported by a global community of contributors.
+{% endblocktrans %}{% endblock %}
+
+{% block body_class %}products-commcare-page tucked-hero{% endblock %}
```

Comment:
Not sure what this does, but I pasted it from the product page.

Reply:
this is important for any custom css you want to do...say in the hero image..